coaster/utils/classes.py

```diff
@@ -146,6 +146,12 @@ class LabeledEnum(six.with_metaclass(_LabeledEnumMeta)):
         >>> NAME_ENUM[NAME_ENUM.THIRD].title
         'Third'
 
+    To make it easier to use with forms and to hide the actual values, a list of (name, title)
+    pairs is available::
+
+        >>> sorted(NAME_ENUM.nametitles())
+        [('first', 'First'), ('second', 'Second'), ('third', 'Third')]
```

Comment:
if you add an `__order__` declaration to the enum, sorting won't be necessary. The unit test below can also remove the PY2 vs PY3 versions.